packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -206,6 +206,30 @@ class RenderSliverMainAxisGroup extends RenderSliver with ContainerRenderObjectM
     }
   }
 
+  @override
+  double? childScrollOffset(RenderObject child) {
+    assert(child.parent == this);
+    final GrowthDirection growthDirection = constraints.growthDirection;
+    switch (growthDirection) {
+      case GrowthDirection.forward:
+        double childScrollOffset = 0.0;
+        RenderSliver? current = childBefore(child as RenderSliver);
+        while (current != null) {
+          childScrollOffset += current.geometry!.scrollExtent;
+          current = childBefore(current);
+        }
+        return childScrollOffset;
```

Comment:
Should this value be available through the childMainAxisPosition method that already exists?

Reply:
It might not work, `childMainAxisPosition` is calculated by `calculatePaintOffset`, which may not be accurate for invisible elements.

Reply:
Ah I see, thanks for clarifying